src/ui/units/dash/containers/Dialogs/Settings/Settings.tsx

```diff
@@ -121,7 +122,10 @@ const Settings = () => {
             dispatch(
                 setSettings({
                     ...settings,
-                    autoupdateInterval: autoupdateInterval || null,
+                    autoupdateInterval:
+                        (typeof autoupdateInterval === 'string'
+                            ? parseInt(autoupdateInterval)
+                            : autoupdateInterval) || null,
```

Comment:
Do we really need fallback to null?
I don't like that if we have false in settings we will get null instead